src/colmap/scene/database.cc

```diff
@@ -441,6 +449,12 @@ std::vector<Image> Database::ReadAllImages() const {
 
   SQLITE3_CALL(sqlite3_reset(sql_stmt_read_images_));
 
+  for (auto& image : images) {
```

Comment:
nit: Alternatively, this could be more efficiently done by implementing a ReadAllPosePriors() method?

Reply:
We now access separately the pose priors in the database :+1: